### DISCUSSION TITLE
Scaffolding tool: any way to not generate InitDataContext()?

### DISCUSSION THREAD

**Discussion by shameleo** [linq2db/linq2db#4648] [General]
My case: two data contexts are generated as partial classes to be merged into one (for cross database queries). As now two `partial void InitDataContext();` exists I get compilation error. Need to manually remove it from generation, deal with messed up formatting and git diffs as autoformatting is applied on save. And newbies can be confused after regeneration.
So not big deal, but annoying.
Didn't found any option to disable it.

**Comment by MaceWindu** [comment]
Yes, there is `--add-init-context false` option for that

**Reply by shameleo** [reply]
Sorry, I've looked through outdated help output saved in file. It works.
